tests/integration/dataframe/test_dataframe.py

```diff
@@ -6,18 +6,30 @@
 import responses
 from nisystemlink.clients.core import ApiException
 from nisystemlink.clients.dataframe import DataFrameClient
-from nisystemlink.clients.dataframe import models
-
-basic_table_model = models.CreateTableRequest(
-    columns=[
-        models.Column(
-            name="index",
-            data_type=models.DataType.Int32,
-            column_type=models.ColumnType.Index,
-        )
-    ]
+from nisystemlink.clients.dataframe.models import (
```

Comment:
The `models` namespace is making the tests unnecessarily verbose, so I converted it to individual imports.